src/outlier_detection/auto_outliers.py

```diff
@@ -249,7 +249,7 @@ def run_auto_flagging(
 
 def apply_short_form_filters(
     df: pd.DataFrame,
-    form_type_no: str = "0006",
+    form_type_no: str = "",
```

Comment:
not sure about this, do you not think we could keep in the default value of 0006?

Reply:
This must have been a typo, I dont remember changing it unless I brought it in when I did git merge origin/develop...